script.js

```diff
@@ -0,0 +1,602 @@
+                button.classList.add("key");
+                button.classList.add("ArrowDown");
+            }else if(englishKeysUp[key] === "&#8594"){
+                button.classList.add("key");
+                button.classList.add("ArrowRight");
+            }else if(englishKeys[key] == "Backspace"){
+                button.classList.add("big");
+                button.classList.add("backpace");
+                button.addEventListener("click", () => {
+                    textarea.setRangeText('', textarea.selectionStart - 1, textarea.selectionEnd);
+                    textarea.focus();
+                  });
+            }else if(englishKeys[key] == "CapsLock"){
+                button.classList.add("big");
+                button.classList.add("capslock");
+            }else if(englishKeys[key] == "Shift"){
+                button.classList.add("big");
+                button.classList.add("shift");
+                button.addEventListener("click", () => {
+                    shift = !shift;
+                    upperCase(lang)
+                    textarea.focus();
+                })
+            }else if(englishKeys[key] == "Enter"){
+                button.classList.add("big");
+                button.classList.add("enter");
+                button.addEventListener("click", () => {
+                    textarea.value += "\n";
+                    textarea.focus();
+                  });
+            }else if(englishKeys[key] == "Tab"){
+                button.classList.add("big");
+                button.classList.add("tab");
+                button.addEventListener("click", () => {
+                    textarea.value += "   ";
+                    textarea.focus();
+                })
+            }else if(englishKeys[key] == "Space"){
+                button.classList.add("space");
+                button.addEventListener("click", () => {
+                    textarea.value += " ";
+                    textarea.focus();
+                })
+            }else if(englishKeys[key] == "Control"){
+                button.classList.add("control");
+            }else if(englishKeys[key] == "Delete"){
+                button.classList.add("delete");
+                button.addEventListener("click", () => {
+                    textarea.setRangeText('', textarea.selectionStart, textarea.selectionEnd + 1);
+                    textarea.focus();
+                  });
+            }else if(englishKeys[key] == "Alt"){
+                button.classList.add("alt");
+            }else{
+                button.classList.add("key");
+                button.addEventListener("click", () => {
+                    textarea.value += button.innerHTML;
+                    textarea.focus();
+                })
+                
+            }
+            button.innerHTML = `${englishKeys[key]}`
+            fragment.append(button);
+        }
+        keyBoard.append(fragment);
+    }else if(l == "rus"){
+        for(let key in russianKeys){

```

Comment:
It can be done withoud code duplication for the each language.
Just create function which will return `fragment` with `button`s, f.e.

```
function getButtons(languageKeys) {
   const buttonsContainer = ...;

   for(...) { ... }

   return buttonsContainer;
}
```